Modules/System/Geolocation/src/GeolocationStatus.Enum.al

```diff
@@ -0,0 +1,44 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Specifies the status of the geographical location data.
+/// </summary>
+enum 50100 "Geolocation Status"
+{
+    Extensible = false;
+
+    /// <summary>
+    /// Available
+    /// </summary>
+    value(0; Available)
+    {
+        Caption = 'Available';
+    }
+
+    /// <summary>
+    /// No data (no data could be obtained).
+    /// </summary>
+    value(1; "No Data")
```

Comment:
When does this status is set? Can it be tested.

Same for `Timed out`.

Reply:
I have added tests for these. They are set by the DotNet Location, so it is tested by mocking these two values.